starboard/nplb/posix_compliance/posix_file_write_test.cc

```diff
@@ -285,6 +286,90 @@ TYPED_TEST(PosixFileWriteTest, PwriteZeroLength) {
   EXPECT_EQ(close(file), 0);
 }
 
+TYPED_TEST(PosixFileWriteTest, BasicWritev) {
+  ScopedRandomFile random_file(0, ScopedRandomFile::kDontCreate);
+  const std::string& filename = random_file.filename();
+  int file =
+      open(filename.c_str(), O_RDWR | O_CREAT | O_TRUNC, S_IRUSR | S_IWUSR);
+  ASSERT_GE(file, 0) << strerror(errno);
+
+  const std::string data1 = "Hello, ";
```

Comment:
I guess we cannot make these constexpr because we're taking its `c_str()`
in l.301 and subsequents?